rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/tests/type.ts

```diff
@@ -0,0 +1,4 @@
+export interface ModelValue {
+  enabled: boolean;
+  datetime: string;
+}
```

Comment:
this type needs to be moved out from the tests as it belongs to the component. To do:

- create "types" folder inside activity folder;
- move the type to a new file named activityTypes;
- rename the type to DateFilterProps;